lib/python/gladevcp/combi_dro.py

```diff
@@ -629,13 +632,11 @@ def get_position(self):
         DTG = the distance to go, will mosltly be 0, as this function should not be used
               while the machine is moving, because of time delays
         '''
-        positions = self._position()
-        if self._ORDER == ["Rel", "Abs", "DTG"]:
-            return positions[1], positions[0], positions[2]
-        if self._ORDER == ["DTG", "Rel", "Abs"]:
-            return positions[2], positions[1], positions[0]
-        if self._ORDER == ["Abs", "DTG", "Rel"]:
-            return positions[0], positions[2], positions[1]
+        if self.diameter:
```

Comment:
Does this scale the position no matter what axis?
I'm probably missing some details but only X axis should get scaled.
If this reinforced somewhere else?

Reply:
You have a Combi_DRO object for each axis. And `diameter` is a member of the Combi_DRO class, so you set it only for the X-axis object.
Here an example for gmoccapy ([go to file](https://github.com/LinuxCNC/linuxcnc/blob/2b07041d04501421cbd56a5fa0810e41beaa2214/src/emc/usr_intf/gmoccapy/gmoccapy.py#L1317-L1334)):
``` python
    def _make_lathe(self):
...
        # if we have a lathe, we will need an additional DRO to display
        # diameter and radius simultaneous, we will call that one Combi_DRO_9, as that value
        # should never be used due to the limit in axis from 0 to 8
        dro = Combi_DRO()
        dro.set_property("name", "Combi_DRO_9")
...
        joint = self._get_joint_from_joint_axis_dic("x")
        dro.set_joint_no(joint)
        dro.set_axis("x")
        dro.change_axisletter("D")
        dro.set_property("diameter", True)
...
```